sunpy/net/helio/tests/test_helio.py

```diff
@@ -286,8 +287,6 @@ def test_select_table(client, monkeypatch):
 
 @pytest.mark.remote_data
 def test_time_query(client):
-    start = '2005/01/03'
-    end = '2005/12/03'
-    table_name = b'rhessi_hxr_flare'
-    res = client.time_query(start, end, table=table_name, max_records=10)
-    assert len(res.array) == 10
+    timerange = a.Time('2005/01/03', '2005/12/03')
+    res = client.search(timerange, table_name=b'rhessi_hxr_flare', max_records=10)
```

Comment:
How much work would it be to remove the byte string for table name?

Reply:
One-two liner, if we want to do it, I kept it untouched so that parameters to previous `query` are same as now `search`.
Though I think we should not use byte string here, since no one will wish to specify a byte string in the query.

Reply:
Yeah, changing that is a breaking change. I am not sure how we would want to handle this. If we add support to do both ways and raise a warning if you use a byte string.

Reply:
Done